packages/expect/src/asymmetricMatchers.ts

```diff
@@ -146,14 +145,15 @@ class ArrayContaining extends AsymmetricMatcher {
 
 class ObjectContaining extends AsymmetricMatcher {
   sample: Object;
+  inverse: boolean;
```

Comment:
`inverse` should be part of `AsymmetricMatcher` it seems?

Reply:
I was placing the inverse individually since the compiler complains about the variable not being initialized. `Property 'inverse' has no initializer and is not definitely assigned in the constructor.`